tony-history-server/buildAndDeploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
```

Comment:
Please add a comment with usage instructions. Looks like this script requires one argument.

Would be good to describe the purpose and usage of this script in the README.md you added, too.